app-editors/hteditor/hteditor-2.1.0.recipe

```diff
@@ -45,7 +52,7 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	runConfigure ./configure
+	runConfigure ./configure --bindir="$commandBinDir"
```

Comment:
Is "--omit-dirs" depreceated?

Reply:
I don't know... I've just had a look at existing recipes, and the above worked, so... :)

Reply:
If build fails, please adjust :)

Reply:
Built succeeded and "ht" appears directly in /bin. Complete and utter success! :)